composer.json

```diff
@@ -15,7 +15,8 @@
         }
     ],
     "require": {
-        "php": "^8.1"
+        "php": "^8.1",
+        "nikic/php-parser": "^4.17"
```

Comment:
`"friendsofphp/php-cs-fixer"` needs to be removed from `"require-dev"` once the test have been updated.